hadoop-ozone/recon/pom.xml

```diff
@@ -24,6 +24,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>ozone-recon</artifactId>
   <properties>
+    <classpath.skip>false</classpath.skip>
```

Comment:
Correct me if I am wrong here, I think the classpath.skip should be set to false for ozone-common module as well, isn't it? I am not able to find the file being updated here.

Reply:
Any particular reason why we don't need the ozone-common classpath file?

Reply:
You are right in that the build used to copy the classpath file for `ozone-common` module to `share/ozone/classpath`:

https://github.com/apache/ozone/blob/91d41a0d1ee276b8f73aa66a0ef8d09a3808d5af/hadoop-ozone/dist/pom.xml#L80-L85

With this PR, classpath file for `ozone-common` is no longer created.  The reason for this is the following:

- Classpath files are used for running Ozone commands.
- The specific classpath file to be used is determined by setting `OZONE_RUN_ARTIFACT_NAME` in `hadoop-ozone/dist/src/shell/ozone/ozone`.
- `ozone-common` is not used for any commands as `OZONE_RUN_ARTIFACT_NAME`.
- Therefore the classpath file is unused.

The last (maybe only) command which had used `ozone-common` as `OZONE_RUN_ARTIFACT_NAME` was `version`, replaced by 8d3817cd87411065450a1ef82678a80e64591239 (HDDS-4758) with `ozone-tools`.

Reply:
oh ok, this makes sense